lib/database/schemas.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+/**
+ * Individual Content Type Schema
+ */
+const content = [
+  {
+    name: 'revision',
+    type: 'increments',
+    index: true,
+  },
+  {
+    name: 'id',
+    type: 'string',
+  },
+  {
+    name: 'created',
+    type: 'timestamp',
+  },
+  {
+    name: 'language',
+    type: 'string',
```

Comment:
Language should be a `select` or point to a taxonomy or some such.
Does _not_ have to be for this pull request...but will need to happen